ouroboros-consensus/demo-playground/TxSubmission.hs

```diff
@@ -73,8 +104,126 @@ command' c descr p =
   Main logic
 -------------------------------------------------------------------------------}
 
-handleTxSubmission :: TopologyInfo -> Mock.Tx -> IO ()
-handleTxSubmission _ _ = fail "TODO: redo handleTxSubmission"
+handleTxSubmission :: forall blk.
+                      RunDemo blk
+                   => DemoProtocol blk
+                   -> TopologyInfo
+                   -> Mock.Tx
+                   -> IO ()
+handleTxSubmission p tinfo mocktx = do
+    topoE <- readTopologyFile (topologyFile tinfo)
+    case topoE of
+         Left e  -> error e
+         Right t@(NetworkTopology nodeSetups) -> do
+
+             case M.lookup (node tinfo) (toNetworkMap t) of
+                  Nothing -> fail "Target node not found."
+                  Just _  -> return ()
+
+             let CoreId nid = node tinfo
```

Comment:
I'd rather have a pattern-match + error to avoid the warning.

Reply:
done